### PR TITLE
Add license

### LINKED ISSUE #19: Add license
The package.json file states that the code is under the MIT license. However, there is no license file.

### PR DESCRIPTION
**Pull request by Victoria-DR**
## What issue is this solving?

Closes #19 

### Description

Adds the MIT license.

## Any helpful knowledge/context for the reviewer?

- Is a re-seeding of the database necessary? No
- Any new dependencies to install? No
- Any special requirements to test? No
  - (e.g., admin perms, alt account, etc.)

### Please make sure you've attempted to meet the following coding standards
- [x] Code has been tested and does not produce errors
- [x] Code is readable and formatted
- [x] There isn't any unnecessary commented-out code

### CODE REVIEW

**Review by aedwardg — APPROVED** [review]
Looks good, but please don't merge until #18 is merged and this has been rebased.

**Review by mikejoh12 — APPROVED** [review]

**Pull request merged**
